tests/archiving/test_import.py

```diff
@@ -72,6 +72,7 @@ def test_import_compressed(self):
         compare_obj = dircmp(path_original, path_import)
         self.assertEqual(len(compare_obj.diff_files), 0)
 
+    @unittest.skip("Imports the same name fails")
```

Comment:
This seems sketchy as hell.

I guess given your changes above, this test now cleanly raises a `ValueError`? But, like, it is still broken, I still want it to fail when it's failing, I don't want to just sweep it under the rug.

Reply:
I agree but I would love to re-introduce it when pack-unpack is fully functional. It's therefore a bit intentional to make you feel uncomfortable when you see `skip`. I can change the message to make sure that people know that this test shouldn't be skipped for a long time.

Reply:
Hmm, making me feel uncomfortable now so that future us can feel comfortable and lazy and just trusting future us to come back, get off our lazy butts, and change something seems unwise. If pack-unpack is not fully functional, that should be clear when and by test failure.

Reply:
Alternatively: the test could `assertRaises(OverlappingJobsError` (or similar), which then _forces_ future-us to come back and fix it once pack-unpack is working as then the test-that-it's-broken test will start failing.

I still would rather leave the broken thing broken as a reminder of its brokenness, but I can tolerate silencing it in a way that will force action later.

Reply:
I mean I'm now actively working on it [here](https://github.com/pyiron/pyiron_base/pull/1440) (although I have to admit that because of the error caused by `test_import` I cannot work on it right now), so it's not like Biden or Trump promising something in their next presidency?